controllers/applicationvolumereplication_controller.go

```diff
@@ -343,8 +343,7 @@ func (r *ApplicationVolumeReplicationReconciler) createVRGClientManifest(name st
 		   "namespace": "%s"
 		},
 		"spec": {
-		   "affinedCluster": "us-central",
-		   "applicationLabels": {
+		   "pvcSelector": {
 			  "matchExpressions": [
```

Comment:
(nit) Can we remove the `matchExpressions` from here as well

Reply:
Sure